docs/content/foundations/layout-positioning/layout-grid.mdx

```diff
@@ -0,0 +1,356 @@
+---
+browser_title: NCI Design System Layout Grid
+page_title: Layout Grid
+nav_label: Layout Grid
+nav_order: 25
+template_type: utility
+---
+
+import GridColExample from '../../../src/components/grid-col-example';
+
+<style>{`
+	.site-code-preview__showcase > .grid-container > .grid-row > div,
+	.site-code-preview__showcase > .grid-row.grid-gap > div > div,
+	.site-code-preview__showcase > .grid-row.grid-gap-lg > div > div,
+	.site-code-preview__showcase > .grid-row:not(.grid-gap, .grid-gap-lg) > div {
+		padding: 16px;
+		border: 1px solid black;
+		background-color: #f0f0f0;
+	}
+	.site-code-preview__showcase > .grid-row,
+	.site-code-preview__showcase > .grid-container > .grid-row {
+		margin-top: 12px;
+	}
+	.site-code-preview__showcase > .grid-row.grid-gap,
+	.site-code-preview__showcase > .grid-row.grid-gap-lg {
+		background-color: #dfe1e2;
+	}
+`}</style>
+
+Use our flexible grid system to structure website content. The grid is mobile-first, powered by flexbox, and based on a 12-column system.
```

Comment:
I don't like the use of "our"  here as the NCIDS uses the USWDS grid system. We should note this.